core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-variants/dot-experiments-configuration-variants.component.html

```diff
@@ -38,13 +38,23 @@ <h2 class="flex align-items-center gap-1 uppercase">
 
                     <span class="separator">|</span>
 
-                    <p-button
-                        data-testId="variant-weight"
-                        label="{{ variant.weight | number : '2.2-2' }}% {{
-                            'experiments.configure.variants.weight' | dm
-                        }}"
-                        styleClass="p-button-sm p-button-link"
-                    ></p-button>
+                    <div
+                        [pTooltip]="'experiment.configure.edit.only.draft.status' | dm"
+                        [tooltipDisabled]="vm.isExperimentADraft"
+                        tooltipPosition="bottom"
+                        data-testId="tooltip-on-disabled"
+                    >
+                        <button
+                            class="p-button-sm p-button-link"
+                            [disabled]="!vm.isExperimentADraft"
+                            data-testId="variant-weight"
+                            label="{{ variant.weight | number : '2.2-2' }}% {{
+                                'experiments.configure.variants.weight' | dm
+                            }}"
+                            pButton
+                            pRipple
+                        ></button>
+                    </div>
```

Comment:
When the Experiment is Running we can edit the variant?

Reply:
new issue for that.